Userland/Games/GameOfLife/Pattern.cpp

```diff
@@ -24,13 +28,11 @@ void Pattern::set_action(GUI::Action* action)
 
 void Pattern::rotate_clockwise()
 {
-    Vector<DeprecatedString> rotated;
-    for (size_t i = 0; i < m_pattern.first().length(); i++) {
-        StringBuilder builder;
-        for (int j = m_pattern.size() - 1; j >= 0; j--) {
-            builder.append(m_pattern.at(j).substring(i, 1));
-        }
-        rotated.append(builder.to_deprecated_string());
-    }
+    Vector<char> rotated;
+    for (size_t i = 0; i < m_width; i++)
+        for (int j = height() - 1; j >= 0; j--)
+            rotated[i * height() + j] = at(i, j);
```

Comment:
Don't skip braces for nested loops like this.
```suggestion
    for (size_t i = 0; i < m_width; i++) {
        for (int j = height() - 1; j >= 0; j--) {
            rotated[i * height() + j] = at(i, j);
        }
    }
```